.github/workflows/ruby.yml

```
@@ -65,10 +83,29 @@ jobs:
+        with:
+          path: ~/docker-ckan-image
+          key: ${{ runner.os }}-ckan-docker-image-${{ steps.ckan-docker-image-hash.outputs.DIGEST }}
       - name: Prepare Ckan
+        if: steps.cache-ckan-images.outputs.cache-hit != 'true'
         run: |
           docker pull ghcr.io/shirasagi/ckan:latest
           docker tag ghcr.io/shirasagi/ckan:latest shirasagi/ckan:latest
+          mkdir -p ~/docker-ckan-image-cache
+          docker save shirasagi/ckan:latest -o ~/docker-ckan-image
+      - name: Load Ckan
+        if: steps.cache-images.outputs.cache-hit == 'true'
```

actionlint (1.7.12)

[error] property "cache-images" is not defined in object type {cache-ckan-images: {conclusion: string; outcome: string; outputs: {string => string}}; ckan-docker-image-hash: {conclusion: string; outcome: string; outputs: {string => string}}; elasticsearch: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)